### PR TITLE
Update release-notes.md for Stripe Payment provider v13.1.0

### PR DESCRIPTION
**Pull request by umbracotrd**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_
=> It's a minor release so no rush on this.

### CODE REVIEW

File: commerce-add-ons/payment-providers/stripe/release-notes.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ In this section, you can find the release notes for each version of Stripe Payme
 
 <summary>Version 13</summary>
 
+#### 13.1.0 (December 28th 2023)
```
Comment (alina-tincas) [inline commerce-add-ons/payment-providers/stripe/release-notes.md:19]: can you please add a link to this as well, similar to https://github.com/umbraco/UmbracoDocs/pull/5737/files? 😊 
image

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @umbracotrd thank you for the PR! 😃 

Can you please add the github link to the release note as well? 😊

File: commerce-add-ons/payment-providers/stripe/release-notes.md (hunk 1 above)
Comment (alina-tincas) [reply commerce-add-ons/payment-providers/stripe/release-notes.md:19]: Hi @umbracotrd I was wondering if you have seen my latest message and if you are able to make add the label for stripe and add a link to it here?  😊

**Comment by alina-tincas** [comment]
Hi @umbracotrd Happy New Year 😊 

I was wondering if you have seen my latest message and if you are able to make the necessary changes?

**Comment by umbracotrd** [comment]
Hi @alina-tincas, I've just added a label and link it to the release notes.

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Awesome, thank you for making the changes #h5YR 

Merging this in 😊

**Pull request merged**